conf/application.conf

```diff
@@ -180,6 +180,22 @@ auditing {
   }
 }
 
+controllers {
+  # Avoid auditing requests for static assets to meet splunk storage requirements
+  # Play's built-in controller for serving static assets  
+  controllers.Assets {
```

Comment:
Nit: a bit verbose. Something like `controllers.Assets.needsAuditing = false`

Same for the other props below.